.travis.yml

```diff
@@ -10,6 +10,8 @@ branches:
     - /^v\d+\.\d+(\.\d+)?(-\S*)?$/
     # … and release branches, e.g. jlc/v0.2.0-release
     - /^.+-release$/
+    # temporary added for merkledb branch
+    - merkledb
```

Comment:
> d1b22a0 is probably to be reverted 📔

Do we need to revert two lines above only? The rest is probably required for building on OSX.

Reply:
Yes, I judged on the commit titles